core/server/worker/src/main/java/alluxio/worker/netty/NettyExecutors.java

```diff
@@ -34,7 +36,7 @@
   public static final ExecutorService ASYNC_CACHE_MANAGER_EXECUTOR =
       new ThreadPoolExecutor(THREADS_MIN,
           Configuration.getInt(PropertyKey.WORKER_NETWORK_NETTY_ASYNC_CACHE_MANAGER_THREADS_MAX),
-          THREAD_STOP_MS, TimeUnit.MILLISECONDS, new SynchronousQueue<>(),
+          THREAD_STOP_MS, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<>(512),
```

Comment:
I suggest we make `512` a configuration property

Reply:
I prefer not to have this configurable as 512 is already a very large number. I don't see the importance for users to tune this parameter